lib/Db/UserDAO.php

```diff
@@ -0,0 +1,106 @@
+<?php
+namespace OCA\Password_Policy\Db;
+
+use OCP\IDBConnection;
+
+class UserDAO {
+
+    private $db;
+
+    public function __construct(IDBConnection $db) {
+        $this->db = $db;
+    }
+
+    public function findAllUsers() {
+        $sql = 'SELECT * FROM oc_users LEFT JOIN oc_password_policy_expiration ON oc_users.uid = oc_password_policy_expiration.uid';
```

Comment:
columns need to be escaped with \` or you use the query builder which does it automatically.